redwood-lazylayout-view/src/main/kotlin/app/cash/redwood/lazylayout/view/ViewLazyList.kt

```diff
@@ -105,22 +108,14 @@ internal open class ViewLazyList private constructor(
       addOnScrollListener(
         object : RecyclerView.OnScrollListener() {
           override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
-            userHasScrolled = true // Prevent guest code from hijacking the scrollbar.
-
-            var min = Int.MAX_VALUE
-            var max = Int.MIN_VALUE
-            for (i in recyclerView.indices) {
-              val position = recyclerView.getChildAdapterPosition(recyclerView[i])
-              min = minOf(min, position)
-              max = maxOf(max, position)
-            }
-            if (min == Int.MAX_VALUE || max == Int.MIN_VALUE) {
-              throw NoSuchElementException()
-            }
-            onViewportChanged?.invoke(
-              min.coerceAtLeast(0),
-              max,
-            )
+            if (scrollState == RecyclerView.SCROLL_STATE_IDLE) return
```

Comment:
This is important! We were getting lots of scroll chatter as the LazyList loaded items

Reply:
linearLayoutManager.findFirstVisibleItemPosition() and linearLayoutManager.findLastVisibleItemPosition() work now!

I'm guessing ti's because of `SCROLL_STATE_IDLE` (I didn't debug into it)